src/components/propertydetails/PropertyDetails.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { useParams, NavLink, Outlet } from "react-router-dom";
+import AllProperties from "../allproperties/allproperties";
+import "./propertydetails.css";
+
+function PropertyDetails() {
+  const { id } = useParams();
+  return (
+    <div className="property-details-container">
+      {AllProperties.filter((ele) => ele.id == id).map((ele, index) => (
```

Comment:
What's ele why not give it a more descriptive name

Reply:
okay.

Reply:
> What's ele why not give it a more descriptive name

okay